.github/workflows/build.yml

```diff
@@ -70,7 +70,7 @@ jobs:
       working-directory: camblet-driver 
       run: |
         sudo modprobe tls
-        sudo modprobe camblet ktls_available=1
+        sudo modprobe camblet dyndbg==_ ktls_available=1
```

Comment:
I like the logs in CI, but OK.

Reply:
It caused a lot of noise without that much added value, but we can move it under some flag later.